client/src/App.js

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from 'react';
+import './App.css'; 
+
+function App() {
+  
+  const [courses, setCourses] = useState([])
+  const serverUrl ='http://localhost:5000/react'
+
+  const getData = async () => {
+          try{
+              const res = await fetch(`${serverUrl}`)
```

Comment:
This here is funny: 
`fetch(\`${serverUrl}\`)`
is actually the same as:
`fetch( serverUrl )`

So in case of no dynamic string construction, the template string is actually not necessary :)